lib/src/container/import.rs

```diff
@@ -125,18 +125,19 @@ async fn fetch_manifest(imgref: &OstreeImageReference) -> Result<(oci::Manifest,
 pub async fn find_layer_tar(
     src: impl AsyncRead + Send + Unpin + 'static,
     blobid: &str,
-) -> Result<(
-    impl AsyncRead,
-    impl Future<Output = Result<impl Read + Send + Unpin + 'static>>,
-)> {
+) -> Result<(impl AsyncRead, impl Future<Output = Result<()>>)> {
```

Comment:
The docstring needs to be update to match the new signature/logic.

Reply:
OK, I cleaned up the related logic here too.